src/renderer/fonts/font_loader.rs

```diff
@@ -54,6 +54,16 @@ pub struct FontKey {
     pub font_selection: FontSelection,
 }
 
+impl FontKey {
+    pub fn default() -> FontKey {
```

Comment:
There’s a trait for this, see https://doc.rust-lang.org/std/default/trait.Default.html:

```rust
impl Default for FontKey {
    fn default() -> Self {
      // …
    }
}
```

Reply:
Thanks!